### PR TITLE
SEO-183463 WPF Missing H1

### PR DESCRIPTION
**Pull request by Yvone-Atieno**
Hi @Aishwarya-Ganesan ,
Have worked on missing H1, please review.
Regards,
Yvone.

### CODE REVIEW

File: wpf/Carousel/How-To/Change-SelectedItem-Position.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ control: Carousel
 documentation: ug
 ---
 
-## Change SelectedItem Position
+# Change SelectedItem Position
```
Comment (gowthamanthavasiyappan) [inline wpf/Carousel/How-To/Change-SelectedItem-Position.md:10]: Update like this - Change SelectedItem Position in WPF Carousel

**Review by gowthamanthavasiyappan — COMMENTED** [review]
Commented

File: wpf/Carousel/How-To/Change-SelectedItem-Position.md (hunk 1 above)
Comment (Yvone-Atieno) [reply wpf/Carousel/How-To/Change-SelectedItem-Position.md]: updated

**Review by gowthamanthavasiyappan — APPROVED** [review]
Approved

**Review by Aishwarya-Ganesan — APPROVED** [review]
fine.

**Pull request merged**